ajax/libs/datatables.net-select/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "datatables.net-select",
+  "filename": "dataTables.select.js",
```

```suggestion
  "filename": "dataTables.select.min.js",
```
A minified version is provided in the package, would it not be better to suggest that by default?

```
tree node_modules/datatables.net-select

node_modules/datatables.net-select
├── License.txt
├── Readme.md
├── js
│   ├── dataTables.select.js
│   └── dataTables.select.min.js
└── package.json

1 directory, 5 files
```